pycds/__init__.py

```diff
@@ -364,7 +365,7 @@ class ObsWithFlags(Base):
 sqlalchemy.event.listen(
     metadata, 'before_create',
     DDL('''
-        CREATE OR REPLACE FUNCTION DaysInMonth(date) RETURNS double precision AS
+        CREATE OR REPLACE FUNCTION crmp.DaysInMonth(date) RETURNS double precision AS
```

Comment:
It's possible that now everything is referring to the correct schema, we don't need to define this function in the ORM as well as natively in the database. Wouldn't that be awesome.